### ISSUE TITLE
install errors

### ISSUE THREAD

**Issue by hrafehi** [gymreklab/GangSTR#28]
Hi,

I am having trouble installing this package. I am wondering if the issue is the compiler version?

This is where is fails:

```
# Install GangSTR
./configure --prefix=$PREFIX || die "Error configuring GangSTR"
make || die "Error compiling GangSTR"

In file included from ../src/enclosing_class.h:24:0,
                 from ../src/likelihood_maximizer.h:24,
                 from ../src/genotyper.h:28,
                 from main_gangstr.cpp:31:
../src/read_class.h:45:34: error: ‘constexpr’ needed for in-class initialization of static data member ‘const double ReadClass::NEG_INF’ of non-integral type [-fpermissive]
   const static double NEG_INF = -100; // TODO make smaller?
                                  ^~~
../src/read_class.h:83:41: error: ‘constexpr’ needed for in-class initialization of static data member ‘const double ReadClass::allele1_weight_’ of non-integral type [-fpermissive]
   const static double allele1_weight_ = 0.5;
                                         ^~~
../src/read_class.h:84:41: error: ‘constexpr’ needed for in-class initialization of static data member ‘const double ReadClass::allele2_weight_’ of non-integral type [-fpermissive]
   const static double allele2_weight_ = 0.5;
                                         ^~~
make[2]: *** [GangSTR-main_gangstr.o] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2

```

Thanks!

**Comment by hrafehi** [comment]
Hi,

I am just wondering if you had a chance to look at this issue yet?

Thanks!

**Comment by nmmsv** [comment]
Hello,
Apologies for my late response. Can you please confirm the system that you are using and your compiler version? I'll try to recreate the issue or look for solutions from there. 
We've tested installation on Mac and few different linux machines (and AWS).
Thanks!

**Comment by hrafehi** [comment]
Thanks for the reply.

I eventually figured it out - the error occurred when using gcc version 6.3.0. When I switched to a much older version (4.8.2), it worked.

Just another minor error I had that is easily fixed - the wget for nlopt and htslib gave an error, and to fix it, I had to specify the output name like this:

wget -O nlopt-2.4.2.tar.gz http://ab-initio.mit.edu/nlopt/nlopt-2.4.2.tar.gz
wget -O htslib-1.8.tar.bz2 https://github.com/samtools/htslib/releases/download/1.8/htslib-1.8.tar.bz2

The error had something to do with the file name being too long.
